css/reveal.css

```diff
@@ -65,13 +65,21 @@ body {
     -webkit-transform: scale(1.3);
             transform: scale(1.3); }
 
+.reveal .slides section span.fragment.grow.visible {
+  font-size: 130%;
+}         
+
 .reveal .slides section .fragment.shrink {
   opacity: 1;
   visibility: inherit; }
   .reveal .slides section .fragment.shrink.visible {
     -webkit-transform: scale(0.7);
             transform: scale(0.7); }
 
+.reveal .slides section span.fragment.grow.visible {
```

Comment:
This would be shrink and not grow

Reply:
Yes, thank you for pointing that out. It should be fixed now